daisy_workflows/linux_common/utils/configs.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import shlex
+
+
+def update_grub_conf(original: str, **configs: str) -> str:
```

Comment:
To which OSs this func can apply? Can it fix the hybrid issue?

Reply:
All of the linux translators update the grub config; this new function is only used for SLES at the moment.

> Can it fix the hybrid issue?

Talked offline -- this PR doesn't address any of the findings in b/168671324 with respect to SLES 12.5's hybrid booting.